_includes/meta.html

```diff
@@ -16,19 +16,25 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <!-- Title and meta description
     ================================================== -->
-  <title>{{page.title}}</title>
-  <meta property="og:title" content="{{page.title}}" />
-  <meta name="description" content="{{page.description}}" />
-  <meta property="og:description" content="{{page.description}}" />
+  {% assign pageTitle = title | append: " | " | append: site.title  -%}
+  {%- assign pageDesc = page.description | default: site.description -%}
+  {%- assign pageUrl = site.url | append: page.url  -%}
```

Comment:
Is `site.url` the base url and `page.url` the path?

Reply:
Yes, that's it.